enelog.c

```diff
@@ -40,12 +57,107 @@ usage(void)
                 "  -E             Show energy field in outputs\n"
                 "  -H             Show field headers in outputs\n"
                 "  -h             Show this help message and exit\n"
+                "  -I             Enable IPMI power measurement\n"
 #ifndef NO_NVML
                 "  -g             Enable GPU power measurement\n"
                 "  -G             Show all powers of GPU devices\n"
 #endif
         );
 }
+//ipmi
```

Comment:
Insert a blank line

Reply:
// ipmi
 Unnecessary or meaningless code comments should be deleted.
Or add a more detailed explanation.